api/client/event-stream/source/source.go

```diff
@@ -73,7 +73,7 @@ func (r *rudderSourceStore) Delete(ctx context.Context, sourceId string) error {
 }
 
 func (r *rudderSourceStore) GetSources(ctx context.Context) ([]EventStreamSource, error) {
-	data, err := r.client.Do(ctx, "GET", prefix, nil)
+	data, err := r.client.Do(ctx, "GET", "/v2/sources", nil)
```

Comment:
Even `GetSources` should use the newer endpoint. But I'll update it in the tracking plan PR